app/services/overview/OverviewDataService.scala

```diff
@@ -20,7 +20,8 @@ class OverviewDataService @Inject()(client: ElasticClient) {
       "_cluster/settings",
       "_aliases",
       "_cluster/health",
-      s"_nodes/_all/os,jvm?human=true"
+      "_nodes/_all/os,jvm?human=true",
+      "_all/_settings/index.lifecycle.indexing_complete"
```

Comment:
it this is enough (it doesn't return closed indices for example) then we're game